var/spack/repos/builtin/packages/touchdetector/package.py

```diff
@@ -56,6 +54,10 @@ class Touchdetector(CMakePackage):
     depends_on('libxml2', when='@:4.4')
     depends_on('zlib', when='@:4.4')
 
+    depends_on('morphio@2.0.8:', when='@4.5:5.1')
+    depends_on('mvdtool@2.1.0:', when='@5.1.1:5.6.0')
```

Comment:
I had the review approved, but then I noticed the `5.6.0` here and in line 47. Shouldn't it be `5.5.9999`?

Reply:
(Sorry for approving and dismissing the review, just trying to be careful here)

Reply:
You're right. Keep forgetting about in/exclusive ranges

Reply:
Yeah... It happens :). Ty for taking care of it, approved!